core/templates/dev/head/pages/exploration_editor/ExplorationSaveService.js

```diff
@@ -82,8 +82,9 @@ oppia.factory('ExplorationSaveService', [
           'post_publish_modal_directive.html'),
         backdrop: true,
         controller: [
-          '$scope', '$uibModalInstance', 'ExplorationContextService',
-          function($scope, $uibModalInstance, ExplorationContextService) {
+          '$scope', '$window', '$uibModalInstance', 'ExplorationContextService',
+          function($scope, $window, $uibModalInstance,
```

Comment:
Match the line breaks for these two sets of dependencies, so that they're easier to compare with each other. Things can break if a dependency is missing or misspelled in either one of the lists.

Also, if the contents of (...) span more than one line, break after the first paren and indent the line after that.

Reply:
Done .